.gitignore

```diff
@@ -1,5 +1,12 @@
+# Define binary
+define
+
 # Vendor Directory
 /vendor
 
 # Build Directory
 /.tmpbuild
+
+# IntelliJ
+*.iml
+.idea/
```

Comment:
Personal tooling and editor-specific ignores should be in your personal "global gitignore". Adding them to a specific project isn't realistically maintainable.